fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -1045,21 +1035,30 @@ protected void allocateTenant() {
     }
 
     /**
-     * Populate all the static tables we need
-     *
-     * @param gen
-     * @param tenantKey
+     * populates for the given tenantId the RESOURCE_TYPE table.
+     * 
+     * @implNote if you update this method, be sure to update
+     *           DerbyBootstrapper.populateResourceTypeAndParameterNameTableEntries
+     *           and DerbyFhirDatabase.populateResourceTypeAndParameterNameTableEntries
+     *           The reason is there are three different ways of managing the transaction.
+     * @param tenantId the mt_id that is used to setup the partition.
+     *                 passing in null signals not multi-tenant.
      */
-    protected void populateStaticTables(FhirSchemaGenerator gen, String tenantKey) {
-        IDatabaseAdapter adapter = getDbAdapter(connectionPool);
+    protected void populateResourceTypeAndParameterNameTableEntries(Integer tenantId) {
         try (ITransaction tx = TransactionFactory.openTransaction(connectionPool)) {
-            try {
-                // Very important. Establish the value of the session variable so that we
-                // pass the row permission predicate
-                Db2SetTenantVariable cmd = new Db2SetTenantVariable(adminSchemaName, tenantName, tenantKey);
-                adapter.runStatement(cmd);
```

Comment:
I guess this wasn't so important, after all?

Reply:
Yes, since I removed the method